src/makeRobot.js

```diff
@@ -39,6 +39,70 @@
  */
 function makeRobot(name, wheels, version) {
   // write code here
+  const robot = {
+    name,
+    wheels,
+    version,
+    coords: {
+      x: 0,
+      y: 0,
+    },
+
+    get info() {
+      return `name: ${name}, chip version: ${version}, wheels: ${wheels}`;
+    },
+    get location() {
+      return `${name}: x=${this.coords.x}, y=${this.coords.y}`;
+    },
+
+    goForward(number) {
+      if (number === undefined) {
+        this.coords.y += 1;
+      } else if (number >= 0) {
+        this.coords.y += number;
+      }
+
+      return this;
+    },
```

Comment:
you cat put default value for `number`
```suggestion
    goForward(number = 1) {
      this.coords.y += number;

      return this;
    },
```

Reply:
please fix all cases